Syncplify's Custom messages

fclairamb opened the pull request:
Taking over from #422

fclairamb commented:
So @syncplify & @drakkan, what do you think about this?

This seems like a nice feature to add and a good time to create a new release.

syncplify commented:
I agree. Apologies for mixing in the second change which should have been its own PR. You're right to drop that. I'll do a new and detailed PR once I have time to explore the observation @drakkan made about that. But the primary feature of this PR is highly desirable so I'd be very happy to see it included in your next release. Thanks.

fclairamb commented:
To explain the little pull requests and branches I have pushed...

I have invested little time in it and faced a few issues.
- I made a working version and added this PR.
- The next day, I changed core parts of the testing code and faced some race conditions
- One week later (now) I tried to revert the code but now I have a problem with golangci-lint that won't even valide the same code push... And now I'm not sure how to proceed.

So now I try to understand why we have these differences:
- [working commit 0aa7a8a](https://github.com/fclairamb/ftpserverlib/actions/runs/8041347938])
- [failing commit 0aa7a8a](https://github.com/fclairamb/ftpserverlib/actions/runs/8117758431)

I suspect it's because of the golangci-lint's cache...
